project.clj

```diff
@@ -1,40 +1,45 @@
-(defn- with-ns [n] (str "org.broadinstitute.firecloud-ui." n))
-
 (defproject org.broadinstitute/firecloud-ui "0.0.1"
   :dependencies
   [
-   [dmohs/react "0.2.11"]
-   [org.clojure/clojure "1.7.0"]
-   [org.clojure/clojurescript "1.7.228"]
+   [dmohs/react "1.0.2+15.0.2"]
+   [org.clojure/clojure "1.8.0"]
+   [org.clojure/clojurescript "1.9.293"]
    [inflections "0.9.14"]
    [cljsjs/moment "2.9.0-3"]
    [cljsjs/codemirror "5.10.0-0"]
    [cljsjs/marked "0.3.5-0"]
    ]
-  :plugins [[lein-cljsbuild "1.1.2"] [lein-figwheel "0.5.0-5"] [lein-resource "15.10.2"]]
-  :profiles {:dev {:dependencies [[binaryage/devtools "0.8.2"]]
-                   :cljsbuild
-                   {:builds {:client {:source-paths ["src/cljsdev"]
-                                      :figwheel true
-                                      :compiler
-                                      {:main ~(with-ns "dev")
-                                       :optimizations :none
-                                       :source-map true
-                                       :source-map-timestamp true}}}}
-                   :figwheel {:http-server-root ""}}
+  :plugins [[lein-cljsbuild "1.1.4"] [lein-figwheel "0.5.8"] [lein-resource "16.9.1"]]
+  :profiles {:dev
+             {:dependencies [[binaryage/devtools "0.8.3"]]
+              :target-path "resources/public"
+              :clean-targets ^{:protect false} [:target-path]
+              :cljsbuild
+              {:builds
+               {:client
+                {:source-paths ["src/cljs/test"]
+                 :figwheel true
+                 :compiler
+                 {;; Use this namespace (which requires main) so that testing is readily available
+                  ;; in all dev builds.
+                  :main "broadfcuitest.testrunner"
+                  :optimizations :none
+                  :output-dir "resources/public/build"
+                  :asset-path "build"
+                  :output-to "resources/public/compiled.js"
+                  :source-map true
+                  :preloads [devtools.preload]
+                  :external-config {:devtools/config {:features-to-install
+                                                      [:formatters :hints]}}}}}}}
              :deploy {:cljsbuild
-                      {:builds {:client {:source-paths ["src/cljsprod"]
-                                         :compiler
-                                         {:main ~(with-ns "render")
-                                          ;; As of 10/29/15, advanced optimization triggers
+                      {:builds {:client {:compiler
+                                         {;; As of 10/29/15, advanced optimization triggers
```

Comment:
does `clean` in the `:deploy` profile still work? I'm getting errors locally, but it could be me.